client/models.js

```diff
@@ -1,4 +1,6 @@
-import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor } from './helpers';
+import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor,
+         formatCurrency, getCurrencySymbol, currencyExists } from './helpers';
+import { store } from './store';
```

Comment:
The same problem rose with the unknown type for the Operation constructor in this file (solution: pass a supplementary arg to the ctor); or to the current locale in the store (solution: store a global variable in the helpers, export a setter for this global variable, call the setter from the store quite early (during setupKresus), and then reuse the global variable). Up to you ;)